CarePartner/Views/FolloweeListView.swift

```diff
@@ -181,7 +181,10 @@ struct FolloweeListView: View {
     
     private var followedAccountsList: some View {
         ForEach(Array(manager.followees.values)) { followee in
-            FolloweeStatusView(followee: followee)
+            FolloweeStatusView(
+                followee: followee,
+                initiallyExpanded: manager.followees.count == 1
```

Comment:
Why do you set the count to 1 here? Sorry, not familiar with FolloweeStatusView

Reply:
`initiallyExpanded` is a `Bool` that will either make the followee card expanded or collapsed when the view first appears. The logic requested in [CPA-48](https://tidepool.atlassian.net/browse/CPA-48) asks for the card to be expanded if there's just a single followee, otherwise have all of them collapsed.